lms/djangoapps/instructor_task/api.py

```diff
@@ -276,3 +277,15 @@ def submit_calculate_students_features_csv(request, course_key, features):
     task_key = ""
 
     return submit_task(request, task_type, task_class, course_key, task_input, task_key)
+
+
+def submit_stud_forums_usage_task(request, course_key):
```

Comment:
Hm, the other task functions in this file are longer than this. @stvstnfrd is the pylint error worth truncating student here?